src/funfuzz/js/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+module.exports = {
+  "parserOptions": {
+      "ecmaVersion": 2018,
+  },
+  "rules": {
+    // Adapted from m-c rev 440407:2d2dee08739f (Fx64) tools/lint/eslint/.eslintrc.js
+    "camelcase": "error",
+    "curly": ["error", "multi-line"],
```

Comment:
I personally don't like this as it would require you to re-add curly braces if the condition expands beyond a single line.

Reply:
I've updated the rule to be the same as the one [here](https://github.com/pyoor/DOMfuzz2/blob/5038b31fdb62eb3bb44bfe94523c6cc771a58934/.eslintrc.json#L48).